account_invoice_import/account_invoice_import_config.py

```diff
@@ -46,6 +46,8 @@ class AccountInvoiceImportConfig(models.Model):
         domain=[('type_tax_use', 'in', ('all', 'purchase'))])
     static_product_id = fields.Many2one(
         'product.product', string='Static Product')
+    product_mapping = fields.One2many(
```

Comment:
I really prefer that relational fields are defined with `_id` and in this case `_ids`.

Reply:
Done

Reply:
Agreed. It was a long time ago that I thought things were going the other way with the introduction of the new API, and this may have set some bad examples in our code but the id/ids suffixes are still conventional.